clients/src/main/java/org/apache/kafka/common/ElectionType.java

```diff
@@ -18,10 +18,11 @@
 package org.apache.kafka.common;
 
 import java.util.Arrays;
+import java.util.Set;
```

Comment:
rearranging the imports to match the style might make sense here, given that there are only two import statements.